package.json

```diff
@@ -20,7 +20,8 @@
     "build:cjs": "rollup -c rollup-cjs.config.js",
     "build:iife": "rollup -c rollup-iife.config.js",
     "build:iife-min": "rollup -c rollup-iife-min.config.js",
-    "report-coverage": "./node_modules/.bin/codecov"
+    "report-coverage": "./node_modules/.bin/codecov",
+    "postpublish": "PACKAGE_VERSION=$(cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[\",]//g' | tr -d '[[:space:]]') echo VERSION: $PACKAGE_VERSION && git tag $PACKAGE_VERSION && git push --tags &&  for i in dist/*; do aws s3 cp $i s3://js.radar.io/$PACKAGE_VERSION/; done && echo do not forget to update README"
```

Comment:
not sure if the trailing `%` would end up causing problems:
<img width="953" alt="Screen Shot 2020-07-01 at 1 45 52 PM" src="https://user-images.githubusercontent.com/58698757/86275388-32861c00-bba1-11ea-8568-e5083ed2e2f4.png">